stboot/make_initramfs.sh

```diff
@@ -49,19 +50,26 @@ if "${develop}" ; then
     GOPATH="${gopath}" u-root -build=bb -uinitcmd=stboot -o "${dir}/${initramfs_name}" \
     -files "${dir}/include/${var_file}:etc/${var_file}" \
     -files "${dir}/include/netsetup.elv:root/netsetup.elv" \
+    -files "${dir}/include/start_cpu.elv:/start_cpu.elv" \
+    -files "${key_dir}/cpu_keys/ssh_host_rsa_key:etc/ssh/ssh_host_rsa_key" \
+    -files "${key_dir}/cpu_keys/cpu_rsa.pub:key.pub" \
```

Comment:
we have so many keys around here, maybe something like `cpu_cmd_key.pub`

Reply:
however the names will be, make sure they are named the same inside the initramfs.